chapter_linear-regression/linear-regression-scratch.md

```diff
@@ -215,27 +247,57 @@ class SGD(d2l.HyperParameters):  #@save
                     param.grad.zero_()
 ```
 
-```{.python .input  n=9}
+```{.python .input}
 %%tab tensorflow
 class SGD(d2l.HyperParameters):  #@save
     def __init__(self, lr):
         """Minibatch stochastic gradient descent."""
         self.save_hyperparameters()
-    
+
     def apply_gradients(self, grads_and_vars):
         for grad, param in grads_and_vars:
             param.assign_sub(self.lr * grad)        
 ```
 
+```{.python .input  n=8}
+%%tab jax
+class SGD(d2l.HyperParameters):  #@save
+    def __init__(self, lr):
+        """
+        Minibatch stochastic gradient descent.
+        The key transformation of Optax is the `GradientTransformation`
+        defined by two functions, the `init` and the `update`.
+        The `init` initializes the state and the `update` transforms
```

Comment:
Remove ` in code comments (we have updated the style guide to remove them to keep comments shorter)